### PR TITLE
Adding logic to allow cluster-reader to view .operations.* logs

### PR DESCRIPTION
**Pull request by ewolinetz**
@sosiouxme @jcantrill @richm PTAL

### CODE REVIEW

**Comment by ewolinetz** [comment]
[test]

**Comment by richm** [comment]
lgtm

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
tests failed from curator not starting up in time during reinstall test...

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
trying to figure out what caused the exception in the transport client when we are seeding the intiial ACL

**Comment by ewolinetz** [comment]
build didn't complete in time... [test]

File: deployer/conf/elasticsearch-logging.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ logger:
   io.fabric8.elasticsearch: ${PLUGIN_LOGLEVEL}
   io.fabric8.kubernetes: ${PLUGIN_LOGLEVEL}
 
+  io.fabric8.elasticsearch.plugin.acl.DynamicACLFilter: DEBUG
```
Comment (ewolinetz) [inline deployer/conf/elasticsearch-logging.yml]: This is just for testing... getting an warning from our client and not sure what's the source. That is going to be reverted :)

**Comment by ewolinetz** [comment]
@richm not sure what's causing that stack trace, even with logging turned up to debug. We can take it on as tech debt though since things still seem to work.

**Comment by richm** [comment]
@ewolinetz are you talking about this?
```
[2016-10-20 19:50:36,982][WARN ][com.floragunn.searchguard.ssl.transport.SearchGuardSSLNettyTransport] [Jane Kincaid] exception caught on transport layer [[id: 0x912ad6d0, /127.0.0.1:60154 => localhost/127.0.0.1:9300]], closing connection
java.lang.IllegalStateException: Message not fully read (response) for requestId [9], handler [org.elasticsearch.action.TransportActionNodeProxy$1@5ed1980e], error [false]; resetting
```
If so, and if everything else is working, then yes, let's fix this later.  https://github.com/openshift/origin-aggregated-logging/issues/260

**Comment by ewolinetz** [comment]
@richm yes, that is the one. sounds good. [merge]

**Pull request merged**